providers/discord/src/airflow/providers/discord/hooks/discord_webhook.py

```diff
@@ -80,6 +83,13 @@ def __init__(
         avatar_url: str | None = None,
         tts: bool = False,
         proxy: str | None = None,
+        embeds: list[dict[str, Any]] | None = None,
```

Comment:
```suggestion
        *args: Any:
        *,
        embeds: list[dict[str, Any]] | None = None,
```

I would suggest we make them keyword-only arguments. It's hard to reason the order of these arguments once the number grows.